server/mailet/integration-testing/src/main/resources/smtpserver.xml

```diff
@@ -87,7 +87,11 @@
             <handler class="org.apache.james.smtpserver.fastfail.ValidRcptHandler"/>
             <handler class="org.apache.james.smtpserver.CoreCmdHandlerLoader"/>
             {{#hooks}}
-            <handler class="{{hookFqcn}}"/>
+            <handler class="{{hookFqcn}}">
+                {{#hookConfigAsXML}}
+                <MaxRcptCount>{{value}}</MaxRcptCount>
```

Comment:
The key is not necessarily maxRcptCount but needs to be taken from the key of the map.

Please write unit tests for this class using an entry the is not MaxRcptCount...

Reply:
yes of course ! i left it like that temporarily because `<{{key}}>{{value}}</{{key}}>` did not seem to work at the time, but i agree with you